.github/workflows/test.yml

```diff
@@ -0,0 +1,54 @@
+name: Test
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: ['main']
+  push:
+
+jobs:
+  build_and_test:
+    name: Build and Test
+    runs-on: [x64, qemu-host]
+    container:
+      image: ghcr.io/viamrobotics/micro-rdk-canon:latest
```

Comment:
Probably okay for now, but... it looks like you're planning to build multi-arch images in the dockerfile, and it's easier sometimes to tag per architecture, instead of just "latest" here.

Reply:
yep I have to work my way through buildx so I can generate non native arch